module/bdev/lvol/vbdev_lvol_rpc.c

```diff
@@ -1474,16 +1475,26 @@ rpc_bdev_lvol_shallow_copy_status(struct spdk_jsonrpc_request *request,
 		goto cleanup;
 	}
 
-	cluster_index = spdk_blob_get_shallow_copy_copied_clusters(src_lvol->blob);
+	copied_clusters = spdk_blob_get_shallow_copy_copied_clusters(src_lvol->blob);
 	total_clusters = spdk_blob_get_shallow_copy_total_clusters(src_lvol->blob);
+	result = spdk_blob_get_shallow_copy_result(src_lvol->blob);
 
 	w = spdk_jsonrpc_begin_result(request);
 
 	spdk_json_write_object_begin(w);
-	spdk_json_write_named_bool(w, "in_progress", total_clusters > 0);
-	if (total_clusters > 0) {
-		spdk_json_write_named_string_fmt(w, "status", "%lu/%lu", cluster_index, total_clusters);
+
+	spdk_json_write_named_string_fmt(w, "progress", "%lu/%lu", copied_clusters, total_clusters);
+	if (result > 0) {
+		spdk_json_write_named_string(w, "state", "none");
```

Comment:
When will this `result` become a positive number?

Reply:
https://github.com/longhorn/spdk/blob/7e86ff9d390020e5efbb5d423f47bca26058d81f/lib/blob/blobstore.c#L307

Reply:
At the beginning it is initialized to 1, so I can check that non operations have been made